packages/ag-charts-community/src/chart/series/hierarchy/hierarchySeries.ts

```diff
@@ -150,11 +150,7 @@ export abstract class HierarchySeries<S extends SeriesNodeDatum> extends Series<
     }
 
     getDatumIdFromData(node: HierarchyNode) {
-        const { labelKey } = this;
-
-        if (labelKey != null) {
-            return node.datum?.[labelKey];
-        }
+        return `${node.index}`;
```

Comment:
We should keep the `labelKey` case (with fallback to `node.index`) since it will be less fragile than the `node.index` when there are data-changes?

Reply:
Discussed offline, we'll revisit once we get to Sunburst animation cases.